lib/collection/property-list.js

```diff
@@ -130,7 +130,9 @@ _.assign(PropertyList.prototype, /** @lends PropertyList.prototype */ {
             this._postman_listIndexCaseInsensitive && (index = index.toLowerCase());
 
             // if multiple values are allowed, the reference may contain an array of items, mapped to an index.
-            if (this._postman_listAllowsMultipleValues && this.reference.hasOwnProperty(index)) {
+            if (this._postman_listAllowsMultipleValues &&
```

Comment:
If you check MDN docs for [hasOwnProperty](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty#Using_hasOwnProperty_as_a_property_name), it has an example for handling this situation where you call `Object.prototype.hasOwnProperty` directly.

Reply:
okay. I have tried adding a unit test [here](https://github.com/sadakchap/postman-collection/blob/20a38ac9c0c695162f6246ee1441490d91941f90/test/unit/url.test.js#L1330).

 And will call `Object.prototype.hasOwnProperty ` directly instead of checking whether key is callable or not. 😀

Reply:
👍  Testing the query parameter covers the reported bug but let's also add an isolated (unit) test which tests this change only i.e, a unit test for the `PropertyList.insert` method.